doc/requirements.txt

```
@@ -1,9 +1,9 @@
 dc-qiskit-algorithms==0.0.14
-docutils==0.19
-ipykernel==6.22.0
+docutils==0.20.1
+ipykernel==6.28.0
 jinja2==3.0.3
 nbsphinx==0.9.3
-pennylane==0.30.0
+pennylane==0.33.1
```

```suggestion
pennylane==0.34
```

? Since it'll be released right before anyway